src/services/postService.js

```diff
@@ -18,23 +18,40 @@ const {
 const VIDEO_URLS   = require('../utils/videoUrls');
 
 module.exports = {
+  broadcastAndStorePost: function({ authenticatedUserInstance, permLink, title, body, tags }) {
+    return authenticatedUserInstance.broadcastPost({ permLink, title, body, tags })
+      .then(broadcastSuccess => {
+        return broadcastSuccess
```

Comment:
This ternary is simple enough to be read on 1 line 👍

Reply:
cool, will change to a 1 liner